### DISCUSSION TITLE
(>'-')>   \_('-')_/   <('-'<)

### DISCUSSION THREAD

**Discussion by Diamondking135** [Revolutionary-Games/Thrive#2144] [Q&A] [answered]
Adding a timer for the remains of the ded cells so place dous not fill up with them :3

**Comment by hhyyrylainen** [comment, marked as answer]
Ah, I see, there is an ideas category here. We don't use this to discuss ideas, so I'll try to close this up.
Anyway dead cells have a death timer after which the entity is removed (once the death animation finishes playing), and cell corpse chunks disappear once they have vented out all of their compounds.
